Packet++/header/DnsLayerEnums.h

```diff
@@ -31,7 +31,7 @@ namespace pcpp
 		DNS_TYPE_MG,
 		/** mail rename domain name record */
 		DNS_TYPE_MR,
-		/** NULL record */
+		/** nullptr record */
```

Comment:
Fix me

Reply:
Yes, I think we should keep this one `NULL`